v5/core/utils_test.go

```diff
@@ -480,3 +480,55 @@ func TestSliceContains(t *testing.T) {
 
 	assert.False(t, SliceContains(nil, "foo"))
 }
+
+func TestGetQueryParam(t *testing.T) {
+	nextURL := "/api/v1/offerings?start=foo&limit=10"
+	next, err := GetQueryParam(&nextURL, "start")
+	assert.Nil(t, err)
+	assert.Equal(t, "foo", *next)
+
+	fqNextURL := "https://acme.com/api/v1/offerings?start=bar&limit=10"
+	next, err = GetQueryParam(&fqNextURL, "start")
+	assert.Nil(t, err)
+	assert.Equal(t, "bar", *next)
+
+	// No query parameter
+	next, err = GetQueryParam(&nextURL, "token")
+	assert.Nil(t, err)
+	assert.Equal(t, (*string)(nil), next)
```

Comment:
```suggestion
	assert.Nil(t, next)
```
Unless there's a good reason for using assert.Equal() in these cases, I think you could just use assert.Nil() instead here and in the other nil checks below.